Pod/Classes/LSTracer.h

```diff
@@ -1,159 +1,85 @@
 #import <Foundation/Foundation.h>
 
-#import <opentracing/OTTracer.h>
 #import "LSSpan.h"
+#import <opentracing/OTTracer.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
-/**
- * The error domain for all OpenTracing-related NSErrors.
- */
-FOUNDATION_EXPORT NSString *const LSErrorDomain;
-/**
- * OTUnsupportedFormat should be used by `OTTracer#inject:format:carrier:` and
- * `OTTracer#extractWithFormat:carrier:` implementations that don't support the
- * requested carrier format.
- */
-FOUNDATION_EXPORT NSInteger LSBackgroundTaskError;
-
-/**
- * An implementation of the OTTracer protocol.
- *
- * Either pass the resulting id<OTTracer> around your application explicitly or use the OTGlobal singleton mechanism.
- *
- * LSTracer is thread-safe.
- *
- * @see OTGlobal
- */
+/// The error domain for all OpenTracing-related NSErrors.
+extern NSString *const LSErrorDomain;
+
+/// OTUnsupportedFormat should be used by `OTTracer#inject:format:carrier:` and
```

Comment:
this comment is 100% wrong... obv the error represents background task failures.